chevah/empirical/tests/normal/test_mockup.py

```diff
@@ -316,8 +316,10 @@ def test_bytes_string_conversion(self):
         with self.assertRaises(UnicodeDecodeError):
             value.decode(encoding='utf-8')
 
-        with self.assertRaises(UnicodeDecodeError):
-            value.decode(encoding='utf-16')
+        # UTF-16 never fails on AIX.
```

Comment:
That was my initial approach: using an invalid size string for UTF-16 decoding so that it will break. I agree that the comment is bad, I will look into it.